spec/controllers/snipets_controller_spec.rb

```diff
@@ -57,6 +58,7 @@
 
   describe "GET #new" do
     it "assigns a new snipet as @snipet" do
+      sign_in  User.new(email: 'landry@spaceinvade.rs', password: '123456', username: 'landry')
```

Comment:
You can pull this line to before filter, or better to shared context, and at the beginning of the spec file just include that context. Maybe you can use this as inspiration (https://www.relishapp.com/rspec/rspec-core/v/2-11/docs/example-groups/shared-context).

We'll check unit tests more after scenarios.